sdk/easm/azure-defender-easm/tests/testcase.py

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import functools
+from devtools_testutils import AzureTestCase, PowerShellPreparer
+from azure.defender.easm import EasmClient
+from azure.identity import InteractiveBrowserCredential
+import datetime
+
+RG='nate-test-rg'
+SUB_ID='59af3bf1-d118-4916-808e-923509b34a94'
+WS='testSdkWorkspace9271133'
+ENDPOINT='eastus.easm.defender.microsoft.com'
```

Comment:
These exposed token are insecure. Please follow https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-for-Test to use new test framework so that you can hide them